elastic/kibana
# [maps] adjust icon size when cluster resolution changes

PR auto adjusts icon size style property when cluster resolution changes to provide a dynamic sizing that makes sense for the size of the clusters.

Steps to test
* Install sample web log data set
* Create new map
* Add "clusters" layer
* Adjust resolution. Notice how icon size style property is adjusted to provide a good default when resolution changes

**Default icon size with low resolution**
<img width="600" alt="Screen Shot 2022-07-18 at 2 57 59 PM" src="https://user-images.githubusercontent.com/373691/179616700-e8794cf1-a10a-475b-8b58-92cba26b9d33.png">

**Default icon size with medium resolution**
<img width="600" alt="Screen Shot 2022-07-18 at 2 58 23 PM" src="https://user-images.githubusercontent.com/373691/179616706-3996eef7-c6b7-41a6-a133-7c382c7e1933.png">

**Default icon size with medium-high resolution**
<img width="600" alt="Screen Shot 2022-07-18 at 3 19 17 PM" src="https://user-images.githubusercontent.com/373691/179619331-85d78b7e-068d-45ca-9b7e-64e4352fe35b.png">

**Default icon size with high resolution**
<img width="600" alt="Screen Shot 2022-07-18 at 2 59 12 PM" src="https://user-images.githubusercontent.com/373691/179616719-da3949b4-bbea-4b94-8994-4751b6f64e9f.png">
